workloads/kube-burner/README.md

```diff
@@ -189,6 +194,18 @@ Each iteration creates the following objects:
 - 20 secrets containing 2048 character random string
 - 10 configMaps containing a 2048 character random string
 
+### Churn
+
+Churning a workload allows you to scale down and then up a percentage of deployments or namespaces **AFTER** the workload has run, but before cleanup (if enabled). It takes in a percentage, **CHURN_PERCENT**, which is the percentage of JOB_ITERATIONS it churn during each churn cycle. After the appropriate number of deployments or namespaces have been churned it will sleep for the remainder of **CHURN_WAIT** seconds if the time it took to churn was less than the **CHURN_WAIT**. This loop continues until the **CHURN_DURATION** is reached.
+
+To churn 20% of your **JOB_ITERATIONS** using the namespace method every 30 seconds for a total duration (the calculated time from when the first churn begins + the **CHURN_DURATION**) of 60 minutes you would utilize these variables:
```

Comment:
I assume that this Churning feature doesn't work with node-density-* workloads, doesn't it?

Reply:
It's not really designed that way today however I could add an option to do a % of the pods created in node-density. Do you want me to go that route or doc it as is today?

Reply:
I think doc it should work fine for now

Reply:
Added doc note